### ISSUE TITLE
Provide speaker embeddings to produce mel

### ISSUE THREAD

**Issue by savinay** [NVIDIA/flowtron#107]
Can we provide our own speaker embeddings to produce mel spectrograms using flowtron rather than use the speaker embeddings generated by flowtron? If yes, how should we normalize those embeddings?

**Comment by rafaelvalle** [comment]
yes, re-train the pre-trained Flowtron LibriTTS2K using your own speaker embeddings, preferably at least on LibriTTS2K.

**Comment by savinay** [comment]
Thanks for your reply! what are the settings to provide my own speaker embeddings? I have trained another model which generates speaker embeddings for the LibriTTS dataset. I would like to use those embeddings to train flowtron model.

**Comment by rafaelvalle** [comment]
there's no pre-written code for loading speaker embeddings externally.
you'll need to change flowtron.py.

**Comment by savinay** [comment]
Thank you very much! that helps.
